README.md

```diff
@@ -1,138 +1,70 @@
-
-<!-- CONTRIBUTING -->
-## Contributing
-
-Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
-
-If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
-Don't forget to give the project a star! Thanks again!
-
-1. Fork the Project
-2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
-3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
-4. Push to the Branch (`git push origin feature/AmazingFeature`)
-5. Open a Pull Request
-
-<!-- MARKDOWN LINKS & IMAGES -->
-<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
-[contributors-shield]: https://img.shields.io/github/contributors/ffxivcode/AutoDuty.svg?style=for-the-badge
-[contributors-url]: https://github.com/ffxivcode/AutoDuty/graphs/contributors
-[forks-shield]: https://img.shields.io/github/forks/ffxivcode/AutoDuty.svg?style=for-the-badge
-[forks-url]: https://github.com/ffxivcode/AutoDuty/network/members
-[stars-shield]: https://img.shields.io/github/stars/ffxivcode/AutoDuty.svg?style=for-the-badge
-[stars-url]: https://github.com/ffxivcode/AutoDuty/stargazers
-[issues-shield]: https://img.shields.io/github/issues/ffxivcode/AutoDuty.svg?style=for-the-badge
-[issues-url]: https://github.com/ffxivcode/AutoDuty/issues
+# AutoDuty
+
+<img align="right" width="150" height="150" src="logo.png">
+
+AutoDuty (AD) is a Dalamud plugin for FFXIV that serves as a tool to assist in the creation and following of paths through dungeons and duties. It's designed to enable automatic looping of instanced content with Duty Support, Trusts, or Squadrons.
+
+Support for it can be found in the [Puni.sh Discord server](https://discord.gg/punishxiv).
+
+_Unlicensed code, which follows [default copyright law](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license)_
+
+# Dungeon Paths
+
+<img align="right" height="250" src="Assets/paths.png">
+
+Each dungeon supported by AutoDuty has a pre-configured path laid out, which should complete the dungeon without any issues. If any issues are encountered, make sure to reach out in the [support Discord](https://discord.gg/punishxiv).
+
+The paths will also run over all loot containers and, when configured, will pick up the loot and do something with it while either in between dungeon runs or after your set amount of runs are over.
+
+Creating paths is also supported and easily accomplished within AutoDuty by using the `Build` tab at the top. For path status, please refer to the [pinned spreadsheet](https://discord.com/channels/1001823907193552978/1236757595738476725/1243059104528994334) in the AutoDuty channel of the [support Discord](https://discord.gg/punishxiv).
```

Comment:
these discord links i would prefer direct to channel links:
ad -> https://discord.com/channels/1001823907193552978/1236757595738476725

Reply:
Resolved in [08d171f](https://github.com/ffxivcode/AutoDuty/pull/317/commits/08d171ffdf912c94222db68a7e03508701797170)